Version 1.2.1/startup

```diff
@@ -1,7 +1,11 @@
 term.clear()
 term.setCursorPos(1, 1)
-shell.run("delete","install")
-shell.run("delete","eject")
-shell.run("cp","disk/install","install")
-shell.run("cp","disk/eject","eject")
+if fs.exsist("/install") then
+  fs.delete("/install")
+end
+if fs.exists("/eject") then
+  fs.delete("/eject")
+end

```

Comment:
These `fs.exists` calls are redundant, `fs.delete` doesn't error if the file couldn't be found.

Reply:
Will be fixed tomorrow.